src/core/Snackbar/Snackbar.tsx

```diff
@@ -0,0 +1,64 @@
+import { Snackbar as MuiSnackbar } from "@material-ui/core";
+import IconButton from "@material-ui/core/IconButton";
+import { SnackbarProps as SnackbarBaseProps } from "@material-ui/core/Snackbar";
+import CloseIcon from "@material-ui/icons/Close";
+import React from "react";
+import { useStyles } from "./styles";
+
+type Variant = "success" | "warning" | "error" | undefined;
+
+export interface SnackbarProps extends SnackbarBaseProps {
+  variant?: Variant;
```

Comment:
May you handle the styling for the case of `variant: undefined`. Either assign some styling as a fallback or make any of the existing options as default